### DISCUSSION TITLE
nsfw sddm

### DISCUSSION THREAD

**Discussion by celikj** [prasanthrangan/hyprdots#1130] [Q&A]
how can I disable anime girls on sddm theme? and also I want to remove that probably japanse  writing

**Comment by kRHYME7** [comment]
If you're using candy theme
 ``` /usr/share/sddm/themes/Candy/backgrounds/ ``` 
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/23e73488-2b0a-4b02-89b1-a95990ebb5cd)

**Comment by mahaveergurjar** [comment]
To remove Chinise's writing edit the  Candy theme.conf
```shell
code  /usr/share/sddm/themes/Candy/theme.conf/  
``` 
![240321_11h21m12s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/d2c448ae-d4d3-4f00-bbe8-f967abd66519)

**Reply by celikj** [reply]
thank you
